extensions/hibernate-reactive/runtime/src/main/java/io/quarkus/hibernate/reactive/runtime/ReactiveSessionProducer.java

```diff
@@ -34,13 +34,24 @@
 
     public void disposeStageSession(@Disposes Stage.Session reactiveSession) {
         if (reactiveSession != null) {
-            reactiveSession.close();
+            //We're ignoring the returned CompletionStage!
```

Comment:
I agree with the change but I think the comment should be:

```suggestion
            //We must not ignore the returned CompletionStage!
```

Reply:
I'm actually referring to the fact that the `disposeStageSession` method itself (the one we're changing here) is returning void - while it *should* chain the event back to a reactive chain. That's sadly not possible with CDI at the moment.

So "must not ignore" would IMO be confusing as we are actually ignoring it here?